packages/slate-editor/src/extensions/galleries/GalleriesExtension.tsx

```diff
@@ -17,16 +19,27 @@ import {
 
 interface Parameters {
     availableWidth: number;
-    onEdit: (editor: Editor) => void;
-    withWidthOption: boolean | undefined;
+    onEdited?: (
+        editor: Editor,
+        gallery: GalleryNode,
+        extra: {
+            successfulUploads: number;
+            failedUploads: Error[];
+        },
+    ) => void;
+    onShuffled?: (editor: Editor, updated: GalleryNode, original: GalleryNode) => void;
+    withMediaGalleryTab: false | { enabled: true; newsroom: NewsroomRef };
```

Comment:
Isn't `enabled: boolean` more appropriate than `enabled: true`? I'm currently in the process of testing features in Prezly by integrating the changes from this PR, and this error appears:

```tsx
<Editor
  withGalleries={{
    withMediaGalleryTab: { newsroom: story.newsroom, enabled: isGalleryEnabled },
  }}
  // ...
/>
```

```
Type 'boolean' is not assignable to type 'true'.ts(2322)
```

Reply:
8b0ade2fed84c27c222fdbaee6d5c3e892b224ea